### PR TITLE
/usr/bin symlink fix 3097

### PR DESCRIPTION
**Pull request by sandrew**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
@wstephenson is your reviewer :dancers: check https://trello.com/c/nUd4cNt0/3097-cnt-suseconnect-binary-goes-missing-after-update 
@wstephenson: Please review this pull request using our guidelines: 
* test for acceptance criteria / functionality 
* check if the new code is covered with tests 
* check for unintended consequences 
* encourage usage of the boyscout rule 
* make sure the code is architected in the best way 
* check that no unnecessary technical debt got introduced 
* make sure that no unnecessary FIXMEs or TODOs got added

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
http://river.suse.de/job/SCC-Connect-Prophet/77237/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project connect on step docker-run-tests.
( Failure: Merged b3f0bd10fad776c00a41572fb508a07d27883455 into a7914602abdde1cd6e8d9fdeddf9d964fdb1e152 )

**Comment by sandrew** [comment]
I pushed a fix for failing scenario. It was failing because of recent changes in SCC API.

In future we should mock up the responses from SCC API here to be independent of it's state and to have more fast tests.

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
http://river.suse.de/job/SCC-Connect-Prophet/77244/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged afc8da919fb5f11ff1a7aa264c893f7e8b3df518 into a7914602abdde1cd6e8d9fdeddf9d964fdb1e152 )

File: package/SUSEConnect.spec [hunk 1]
```diff
@@ -91,6 +91,7 @@ fi
 # remove stale update-alternatives config left by previous split, versioned packaging of SUSEConnect
 if update-alternatives --config SUSEConnect  &> /dev/null ; then
   update-alternatives --quiet --remove-all SUSEConnect
+  ln -fs %{_sbindir}/%{name} %{_bindir}/%{name}
```
Comment (wstephenson) [inline package/SUSEConnect.spec]: This should be a relative symlink, see comment at top of 
/usr/lib/rpm/brp-suse.d/brp-25-symlink (from package brp-check-suse) 
The brp checks run at package build time fix this for us, but relinking in %post will undo the fix in upgraded packages and so the package will likely fail SLES integration review.

```ln -fs ../%{_sbindir}/%{name} %{_bindir}/%{name}```

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
http://river.suse.de/job/SCC-Connect-Prophet/77652/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 873035f1c6e0494114676599dc9c137446fdcdf5 into a7914602abdde1cd6e8d9fdeddf9d964fdb1e152 )

**Comment by wstephenson** [comment]
lgtm

**Pull request merged**